src/Uno.UI.RuntimeTests/IntegrationTests/common/TestServices.KeyboardHelper.cs

```diff
@@ -318,6 +319,18 @@ public static void GamepadDpadDown(UIElement element = null)
 			{
 				PressKeySequence("$d$_GamepadDpadDown#$u$_GamepadDpadDown", element);
 			}
+
+			/// <param name="text">Assuming lowercase text. To add capitalization, use <see cref="PressKeySequence"/></param>
+			public static void InputText(string text, UIElement element = null)
```

Comment:
any explanation on why this was added?
from what ive seen, this doesnt work: no visual change & no event raised